roles/splunk_common/tasks/enable_forwarding.yml

```diff
@@ -63,7 +63,9 @@
     owner: "{{ splunk.user }}"
     group: "{{ splunk.group }}"
   register: index_disabling
-  when: splunk.role != "splunk_license_master"
+  when:
+    - splunk.role != "splunk_license_master"
+    - splunk.role != "deployment_server"
```

Comment:
I believe the role name should be "splunk_deployment_server"

Reply:
Applied changes